Release notes foreman ansible 11 (#1991)
theforeman/foreman-documentation: guides/doc-Release_Notes/topics/foreman.adoc

[id="foreman-headline-features"]
== Headline Features

> === Foreman Ansible plugin

Reviewer: ```suggestion

=== Foreman Ansible plug-in
```

> * A new option to enable the callback plugin for each ansible job template was added.
> * When an ansible job execution results in an error, display the error message in the config report UI.

Reviewer 2: We should capitalize Ansible
```suggestion
* A new option to enable the callback plugin for each Ansible job template was added.
* When an Ansible job execution results in an error, display the error message in the config report UI.
```

But we always write headline features for humans. That means a paragraph or two of text, rather than a list. Feel free to use a bit more space to explain why it's nice.

[id="foreman-headline-features"]
== Headline Features

=== Foreman Ansible plug-in

> ==== New option to enable the callback plugin for Ansible jobs

Reviewer: ```suggestion

==== New option to enable the callback plugin for Ansible jobs
```

> Previously, the callback plugin was enabled only for the `Ansible Roles - Ansible Default` playbook.

Reviewer: ```suggestion

Previously, the callback plugin was enabled only for the `Ansible Roles - Ansible Default` playbook.
```

In this release, a new feature has been added, giving users the ability to enable the callback plugin for each Ansible job template individually.

> ==== Fix Ansible config report 500 error

Reviewer: ```suggestion
==== Fix Ansible config report 500 error

```

[id="foreman-headline-features"]
== Headline Features

=== Foreman Ansible plug-in

==== New option to enable the callback plugin for Ansible jobs

> Previously, the callback plugin was enabled only for the `Ansible Roles - Ansible Default` playbook.

Reviewer: ```suggestion
Previously, the callback plug-in was enabled only for the `Ansible Roles - Ansible Default` playbook.
```

> In this release, a new feature has been added, giving users the ability to enable the callback plugin for each Ansible job template individually.

Reviewer: ```suggestion
In this release, a new feature has been added, giving users the ability to enable the callback plug-in for each Ansible job template individually.
```
